recipes/bitsets/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "bitsets" %}
+{% set version = "0.7.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/26/41/91d45f9818e25a0ce04a725a60639cdcf615772bb825cee8e8f0e706f8cd/bitsets-0.7.16.zip 
```

Comment:
Please use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

You might have to change the `sha256`